### PR TITLE
Add validation if the client's nonce is a past nonce

### PR DESCRIPTION
**Pull request by manasnisar**
## Changes
Add validation if the client's once is a part nonce

## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by dabasov — APPROVED** [review]
.

File: code/go/0chain.net/chaincore/chain/handler.go [hunk 1]
```diff
@@ -1318,6 +1319,18 @@ func PutTransaction(ctx context.Context, entity datastore.Entity) (interface{},
 		return nil, err
 	}
 
+	s, err := sc.GetStateById(sc.GetLatestFinalizedBlock().ClientState, sc.OwnerID())
```
Comment (dabasov) [inline code/go/0chain.net/chaincore/chain/handler.go]: sc.OwnerID() is not correct
it should be txn.FromClient()

**Review by dabasov — CHANGES_REQUESTED** [review]
lgtm

File: code/go/0chain.net/chaincore/chain/handler.go (hunk 1 above)
Comment (manasnisar) [reply code/go/0chain.net/chaincore/chain/handler.go]: Fixed.

**Review by peterlimg — APPROVED** [review]
LGTM

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**